addon/components/file-renderer/template.hbs

```diff
@@ -0,0 +1,2 @@
+<iframe src={{buildMfrUrl download}} width="100%" height=100% scrolling="no" marginheight="0" frameborder="0">
```

Comment:
@pattisdr do we have a use case for building MFR urls outside of this component? Using a template helper seems like unneeded indirection. Since the url is a function of the `download` property of the component I'd rather see it moved to a computed value there. 

E.g.
```javascript
export default Ember.Component.extend({
   layout,
   download: null,
   mfrUrl: Ember.computed('download', function() {
       ...
   })
});
```

And you template can just:
```html
<iframe src={{mfrUrl}} ... 
```